app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -1022,6 +1021,42 @@ open class BrowserActivity : DuckDuckGoActivity() {
             Timber.e(ex)
         }
     }
+
+    private fun bindMockupToolbars() {
```

Comment:
the new omnibar is higher than the current one, so we need to account for that and create a new mockup toolbar that appears when the app starts.